xbmc/pvr/PVRManager.cpp

```diff
@@ -1647,18 +1650,21 @@ std::string CPVRManager::GetPlayingTVGroupName()
   return IsStarted() && m_guiInfo ? m_guiInfo->GetPlayingTVGroup() : "";
 }
 
-void CPVRManager::UpdateLastWatched(CPVRChannel &channel)
+void CPVRManager::UpdateLastWatched(const CPVRChannelPtr &channel)
 {
+  if (!channel)
```

Comment:
I changed C++ references to references to shared_ptrs. A C++ reference cannot be NULL, but a derefed shared_ptr can. That's why I introduced the addidional checks. But yes, assert might be enough here.